tests/parker.js

```diff
@@ -12,7 +12,7 @@ function generateMetrics(file, metricsArray) {
     {
       name: 'Stylesheet size',
       benchmark: 150000,
-      threshold: 380000,
+      threshold: 430000,
```

Comment:
Adding more themes to side navigation increases the size of CSS noticeably. 
It's OK, because in the end this additional CSS is quite repetitive and will GZIP nicely, so doesn't affect performance too much.

But we need to increase our allowed thresholds so that the tests pass.